clm_ros_wrapper/GazeDirection[] directions
